src/main/resources/db/migration/h2/V1.7.0__create-table-whitelist-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `whitelisted_domains` (
+  `id` mediumint(8) unsigned NOT NULL AUTO_INCREMENT,
+  `domain` varchar(100) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `UK_domain` (`domain`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
Can you take these changes and put into the V1.1__data.sql file in the h2 directory?

While you're at it can you do the same for V1.2__oauthcode.sql?

Helps to keep things tidy.

Reply:
Not repeating work, originally discussed and was decided to create v1.7.0